components/eamxx/src/physics/mam/eamxx_mam_microphysics_process_interface.cpp

```diff
@@ -31,8 +27,6 @@ std::string MAMMicrophysics::name() const {
 void MAMMicrophysics::set_grids(const std::shared_ptr<const GridsManager> grids_manager) {
   using namespace ekat::units;
 
-  logger.trace("entering MAMMicrophysics::set_grids");
```

Comment:
Note: you can still log stuff from this process, using the base class protected method `log`:
```
this->log(LogLevel::trace,"entering MAMMicrophysics::set_grids");
```